src/Arcus.Scripting.KeyVault/Scripts/Set-AzKeyVaultSecretFromFile.ps1

```diff
@@ -0,0 +1,15 @@
+# Use this script to upload a certificate as plain text (multiline-support) into Azure Key Vault.
+
+param (
+    [string][Parameter(Mandatory=$true)] $FilePath = $(throw "The path to the file is required."),
+    [string][Parameter(Mandatory=$true)] $KeyVaultName = $(throw "The path to the file is required."),
+    [string][Parameter(Mandatory=$true)] $SecretName = $(throw "The path to the file is required.")
+)
```

Comment:
Could be useful to allow setting the expiration date on the certificate through `-Expires`

Reply:
Yes, added that and added a test too.